modules/java_api/build.gradle

```diff
@@ -0,0 +1,28 @@
+plugins {
```

Comment:
Is this file related to this PR?

Reply:
This file is needed to pass security checks required to release the plugin.